### PR TITLE
EZP-25635: Added rewrite rule for assets folder

### PR DESCRIPTION
**Pull request by andrerom**
Adopt convention from ezplatform-demo.

Issue: https://jira.ez.no/browse/EZP-25635

### CODE REVIEW

File: doc/apache2/vhost.template [hunk 1]
```diff
@@ -93,8 +93,9 @@
         # More info here : http://en.wikipedia.org/wiki/P3p
         RewriteRule ^/w3c/p3p\.xml - [L]
 
-        # The following rule is needed to correctly display assets from eZ Platform / Symfony bundles
+        # The following rule is needed to correctly display assets from bundles, and project assets
```
Comment (bdunogier) [inline doc/apache2/vhost.template]: "display bundles and project assets" ?

**Comment by bdunogier** [comment]
+1

File: doc/apache2/vhost.template (hunk 1 above)
Comment (gggeek) [inline doc/apache2/vhost.template]: then "bundle and project assets" - the singular makes it slightly easier to understand who goes where :-)

**Comment by andrerom** [comment]
Updated, will create issue and rebase for that, looks ok @dpobel ?

*Side: would potentially be nice to configure asset:install and assetic:dump to use this folder as well so we can get rid of permission rules to write to whole web dir.*

**Comment by bdunogier** [comment]
> Side: As future possible follow up

JIRA ? :-)
I'm not sure I understand the link to permissions, btw.

**Comment by andrerom** [comment]
> I'm not sure I understand the link to permissions, btw.

We currently need to give write permission to whole web folder, it would be nice to restrict that, but part of the sf 3.0 topic for later I guess.

**Comment by dpobel** [comment]
+1

**Comment by bdunogier** [comment]
+1

**Pull request merged**